### PR TITLE
cli: add debug callout in exec

### PR DESCRIPTION
**Pull request by craig-osterhout**
Added callout to `docker exec` CLI reference page to introduce Docker Debug.

Preview: https://deploy-preview-19423--docsdocker.netlify.app/engine/reference/commandline/container_exec/

ENGDOCS-1991

### CODE REVIEW

File: content/engine/reference/commandline/container_exec.md [hunk 1]
```diff
@@ -9,6 +9,14 @@ aliases:
 layout: cli
 ---
 
+> **Introducing Docker Debug**
+>
+> Dive into Docker Debug, a replacement for debugging with `docker exec`. Access
```
Comment (joe0BAB) [inline content/engine/reference/commandline/container_exec.md:14]: It prepend something like: "To easily get a debug shell into any container, we created a new command `docker debug`." And then proceed with "It's a replacement..."

-> put the problem that most people have when they search for docker exec at the beginning (which is getting a shell into their container)
-> tell them we have created a better tool (this is what you already wrote)

Comment (craig-osterhout) [reply content/engine/reference/commandline/container_exec.md:14]: Thanks, @joe0BAB. How about the wording below?

> To easily get a debug shell into any container, use `docker debug`.
> Docker Debug is a replacement for debugging with `docker exec`. With it, you
> can access any container or image, even slim ones, without modifications.
> Plus, you can bring along your favorite debugging tools in its customizable
> toolbox.

Comment (joe0BAB) [reply content/engine/reference/commandline/container_exec.md:14]: Love this! Let's go with this!

File: content/engine/reference/commandline/container_exec.md [hunk 2]
```diff
@@ -9,6 +9,15 @@ aliases:
 layout: cli
 ---
 
+> **Introducing Docker Debug**
+>
+> To easily get a debug shell into any container, use `docker debug`. Docker
+> Debug is a replacement for debugging with` docker exec`. With it, you can
+> access any container or image, even slim ones, without modifications. Plus,
```
Comment (dvdksn) [inline content/engine/reference/commandline/container_exec.md:16]: Maybe "can get a shell in" is better than "can access" ?

**Review by dvdksn — APPROVED** [review]
LGTM, left an optional drive-by comment

**Review by dvdksn — APPROVED** [review]

**Pull request merged**